addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -240,6 +240,8 @@ $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o
 
 // Input
 $input-bg: $o-white !default;
+$form-check-input-checked-color: $o-brand-lightsecondary !default;
+$form-check-input-checked-border-color: $o-brand-primary !default;
```

Comment:
Not directly related to your change, but why is this file still in legacy/ ?

Reply:
@aab-odoo , we have just to decide the right place where to move it I guess.

By a design perspective these files are "core" (`primary_variables.scss`, `secondary_variables.scss`, `bootstrap_overridden.scss`...), others like `ui.scss` could be moved in "webclient".

Reply:
Yes, but the same variables affects core components too (checkboxes, dropdown, popovers..).

Reply:
If it *affects* them, it's not necessarily a problem: there are core components, and in the webclient, we want to style them the way we want, through variables for instance. However, it is a problem if those core components *use* variables **defined** in those files (you don't want a core component to depend on the existence of something defined in web). But in the latter case, maybe those variables should be defined next to the core component directly.

Reply:
The mechanic/logic of `core` components is independent (I see what you mean), but by an UI perspective it's not true 'cause core components use bootstrap/odoo variables & **classes** all together (*).

Without these styles the components will not be rendered correctly (or return errors)... for me it's an hint that those files should be placed in `core`.
Moreover the way how we customize bootstrap impacts visuals outside `webclient`, placing these files there would be reductive IMO.

Another option could be move "core" SCSS files in a main`/web/static/scss` folder... and all the rest in `webclient`... maybe I like it more actually.


(*): again, we're not bootstrap's super-fan and we actually advocate for a custom CSS framework (it would also avoid conflict with website), but as long as it's included it it makes sense to embrace it fully and correctly, what's the point of maintaining two different design for a checkbox :/